assets/js/modules/analytics/datastore/properties.js

```diff
@@ -105,6 +107,11 @@ const baseActions = {
 		invariant( accountID, 'accountID is required.' );
 
 		const { response, error } = yield fetchCreatePropertyStore.actions.fetchCreateProperty( accountID );
+		if ( error ) {
+			const registry = yield Data.commonActions.getRegistry();
+			yield registry.dispatch( STORE_NAME ).receiveError( error, 'createProperty', { accountID } );
+		}
+
```

Comment:
The `getProperties` resolver further below also needs a manual `receiveError( error, 'getProperties', [ accountID ] )` call.

Reply:
Yep, added.